docs/webapi/waitForNotAttached.mustache

```diff
@@ -0,0 +1,9 @@
+Waits for an element to become not attached to the DOM on a page (by default waits for 1sec).
+Element can be located by CSS or XPath.
+
+```
+I.waitForNotAttached('#popup');
```

Comment:
Thing is it's not `detached` by default, it could never have been attached as well. We can have `waitForStalenessOf` or `waitForNotExists`?